jbmc/src/java_bytecode/java_static_initializers.cpp

```diff
@@ -551,7 +551,7 @@ code_blockt get_thread_safe_clinit_wrapper_body(
   // the static initializer wrapper. Enable this for debugging the symex going
   // through the clinit_wrapper.
   //
-  // java::C::clinit_wrapper()
+  // java::C.<clinit_wrapper>()
```

Comment:
I suppose this will make some test-gen regression tests fail, so we should synchronize with a test-gen PR and mark this one as "Requires TG approval".

Reply:
There's a corresponding TG PR.